### PR TITLE
Closes #55

### LINKED ISSUE #55: add an Error 404 page
I would like to add an error 404 page to the website

### PR DESCRIPTION
**Pull request by AdarshRawat1**
Added Error 404 page.

### CODE REVIEW

File: 404.html [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+                    <li></li>
+                    <li></li>
+                    <li></li>
+                    <li></li>
+                    <li></li>
+                </ul>
+                <div class="ui center aligned text canvas-text">
+                    <h1 class=>ERROR 404!<br>
+                    </h1>
+                    <h3 class="error-msg">THE PAGE YOU ARE LOOKING FOR DOESN'T EXIST.<br>
```
Comment (nemesisdesign) [inline 404.html:69]: do not use all uppercase please

File: 404.html [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+    <script defer src="js/feed.js"></script>
+    <script>
+        $(document).ready(function () {
+            createActivityFeed({ "posts": 5, "loadMore": false });
+        });
+    </script>
+    
+</body>
+
+</html>
```
Comment (nemesisdesign) [inline 404.html:130]: ensure the file ends with a blank line

File: index.html [hunk 3]
```diff
@@ -10,6 +10,10 @@
   <link rel="stylesheet" href="css/semantic.min.css" type="text/css">
   <link rel="stylesheet" href="css/style.css" type="text/css" media="screen">
   <link rel="icon" type="image/x-icon" href="images/favicon.png" />
+  <script>
+if (window.location.status === 404) {
+      window.location.href = "/404.html";
+    }</script>
```
Comment (nemesisdesign) [inline index.html:16]: Google Cloud will take care of showing the 404 page when needed. Please remove this.

**Review by nemesisdesign — CHANGES_REQUESTED** [review]
Please follow the commit message guidelines, eg:

```
[change] Add 404 page #55 

Closes #55
```

File: css/404.css [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+  0% {
+    transform: translateY(0) rotate(0deg);
+    opacity: 1;
+  }
+
+  100% {
+    transform: translateY(-800px) rotate(360deg);
+    opacity: 0.3;
+  }
+}
```
Comment (Aryamanz29) [inline css/404.css:84]: Please add a blank line here.

Comment (Aryamanz29) [inline css/404.css:84]: Please add a blank line here.

**Comment by Aryamanz29** [comment]
Hey @AdarshRawat1! 
Your PR title and commit message still don't follow our commit guidelines.
Also make sure QA checks passes when running `./run-qa-checks` script locally.
Please read : https://openwisp.io/docs/developer/contributing.html#how-to-commit-your-changes-properly

**Comment by AdarshRawat1** [comment]
Sir @Aryamanz29 @nemesisdesign  , I'll make a new pull request with all the necessary corrections. Thank you for your feedback.

**Pull request closed without merging**